.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: node_js
-sudo: false
+services:
+- docker
+sudo: required
 node_js:
 - '4.2'
 script:
 - npm test
+- make docker-build
+deploy:
+  provider: script
+  script: scripts/deploy.sh
```

Comment:
Maybe. I think that offers some advantage in terms of the image being "verified" or some such, yes? Efficiency wise, however, pushing it from CI also makes sense because I feel that the image _building_ properly should be a gate for a successful CI run, just as surely as unit tests are. Ultimately, one could ask: why build twice?

So... I could go either way on this. lmk whichever you think is better.